src/borg/testsuite/platform_freebsd.py

```diff
@@ -0,0 +1,30 @@
+""" Dummy file for now, will eventually contain FreeBSD ACL tests """
```

Comment:
Oh, didn't see this initially:
- either `"""bla bla"""
- or:
```
"""
bla
bla
"""

Reply:
and add a dot at the end.

Reply:
other than that, it's ready for merging.

Reply:
All set 👍